packages/0xcert-scaffold/src/core/messages.ts

```diff
@@ -0,0 +1,91 @@
+  [ProviderIssue.ERC721_METADATA_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_METADATA_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_METADATA_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_METADATA_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_METADATA_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_ENUMERABLE_ZERO_ADDRESS]: 'Provided address cannot be zero address.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_ENUMERABLE_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_ENUMERABLE_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_ENUMERABLE_INVALID_INDEX]: 'There is no asset at provided index.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_ZERO_ADDRESS]: 'Provided address cannot be zero address.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_VALID_NFT]: 'Asset with this id does not exist.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_OWNER_APPROWED_OR_OPERATOR]: 'Sender is neither asset owner, approved or operator.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NOT_ABLE_TO_RECEIVE_NFT]: 'Receiver is not able to safely receive the asset.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_NFT_ALREADY_EXISTS]: 'Asset with provided id already exists.',
+  [ProviderIssue.ERC721_METADATA_ENUMERABLE_INVALID_INDEX]: 'There is no asset at provided index.',
+  [ProviderIssue.XCERT_CAPABILITY_NOT_SUPPORTED]: 'Asset ledger does not have capability for this action.',
+  [ProviderIssue.XCERT_TRANSFERS_DISABLED]: 'Transfers on this asset ledger are currently disabled.',
+  [ProviderIssue.XCERT_NOT_VALID_XCERT]: 'Asset with this id does not exist.',
+  [ProviderIssue.XCERT_NOT_OWNER_OR_OPERATOR]: 'Sender is neither asset owner or operator.',
+  [ProviderIssue.XCERT_INVALID_SIGNATURE]: 'Provided signature is invalid.',
+  [ProviderIssue.XCERT_INVALID_SIGNATURE_KIND]: 'Provided signature kind is invalid.',
+  [ProviderIssue.XCERT_CLAIM_PERFORMED]: 'This order was already performed.',
+  [ProviderIssue.XCERT_CLAIM_EXPIRED]: 'Order has expired.',
+  [ProviderIssue.SAFE_MATH_OVERFLOW]: 'This action caused a math error: overflow.',
+  [ProviderIssue.SAFE_MATH_SUBTRAHEND_GREATER_THEN_MINUEND]: 'This action caused a math error: subtrahend is greater then minuend.',
```

Comment:
[ProviderIssue.SAFE_MATH_SUBTRAHEND_GREATER_THAN_MINUEND]: 'This action caused a math error: subtrahend is greater than minuend.',